case_specification.tex

```diff
@@ -0,0 +1,300 @@
+                
+                \caption{Model Fuel Cycle with \gls{MOX} Reprocessing}
+                \label{diag:fc}
+\end{figure}
+\end{frame}
+
+\begin{frame}
+	\frametitle{First Simulation - EU Nuclear Opearations \textasciitilde 2050}
+	Deployment and Reactor data from IAEA PRIS
+	Reprocessing plant and fabrication plant modeled after French La Hague and MELOX site
+	\cite{schneider_spent_2008, hugelmann_melox_1999}.
+		
+\begin{table}[h]
+	\centering
+	\begin{tabularx}{\textwidth}{bb}
+		\hline
+		Specification & Value \\
+		\hline
+		Simulation Time & 1970-2050 \\ 
+		Reprocessing Capacity & 91.6 MTHM of \gls{UNF} per month \cite{schneider_spent_2008} \\
+		Reprocessing Efficiency & 99.8\% \\
+		Reprocessing Streams & Plutonium and Uranium \\
+		\gls{MOX} Fabrication & \small{9\% Reprocessed Pu + 91\% Depleted U} \\
+		\gls{MOX} Fabrication Throughput & 16.25 MTHM of \gls{MOX} per month  \cite{hugelmann_melox_1999} \\
+		\gls{MOX} Fuel Reprocessing Stage &  Used \gls{MOX} gets reprocessed infinitely. \\  
+		Reprocessed Uranium Usage &  None. Stockpile reprocessed U \\
+		\hline
+	\end{tabularx}
+	\caption {Specification for Historical Operation of \gls{EU} Case}
+	\label{tab:sim_eu}
+\end{table}
+	
+\end{frame}
+
+\begin{frame}
+	\frametitle{Second Simulation - French Transition to SFRs \textasciitilde 2160}
+	
+\begin{table}[h]
+	\centering
+	\begin{tabularx}{\textwidth}{bb}
+		\hline
+		Specification & Value \\
+		\hline
+		Simulation Time & 1970-2160 \\
+		\gls{SFR} Available Year & 2040 \\
+		Reprocessing Capacity & $\infty$ \\
+		Reprocessing and Fabrication Begins & 2020 \\
+		Separation Efficiency & 99.8 \% \\
+		Reprocessing Streams & plutonium and uranium \\
+		\small{Used \gls{UOX} and Depleted U Inventory} & 141,659 MTHM {\small (From first simulation)} \\
+		\small{Additional Used \gls{UOX} or Depleted U} & None  \\
+		\gls{MOX} Fabrication &  \small{11\% Reprocessed Pu + 89\% Depleted U}  \\
+		\gls{MOX} Fabrication Throughput & infinite \\
+		\gls{MOX} Fuel Reprocessing Stage &  Used \gls{MOX} gets reprocessed infinitely. \\
+		Reprocessed Uranium Usage &  None. Stockpile reprocessed U. \\
+		\hline
+	\end{tabularx}
+	\caption {Specification for French Transition to \gls{SFR} case }
+	\label{tab:sim_france}
+\end{table}
+
+\end{frame}
+
+
+\begin{frame}
+	\frametitle{Reactor Specifications - PWR}
+	
+	\begin{table}[h]
+	\centering
+	\begin{tabularx}{\textwidth}{bb}
+		\hline
+		Specification & Value \\
+		\hline
+		PWR Cycle Time & 18 months \\ 
+		PWR Refueling Outage & 2 months \\
+		Fuel Mass per Assembly & 523.4 kg \\
+		Burnup & 51 GWd/tons \\
+		Num. of Aseem. per Core & 193 for 1,000 MWe, linearly adjusted\\
+		Num. of Assem. per Batch & 1/3 of the core \\
+		Fuel & \small{French \glspl{PWR} prefer \gls{MOX} but also accept \gls{UOX}}\\
+		\hline
+	\end{tabularx}
+	\caption {\gls{PWR} Specifications}
+	\label{tab:pwr}
+	\end{table}
+\end{frame}
+
+\begin{frame}
+	\frametitle{Reactor Specifications - BWR}
```

Comment:
Rather than spreading these tables across many slides... Why not a single table with all of the reactors getting their own column?